src/services/completions.ts

```diff
@@ -559,16 +559,6 @@ namespace ts.Completions {
         }) || { type: "none" };
     }
 
-    function getSymbolName(symbol: Symbol, origin: SymbolOriginInfo | undefined, target: ScriptTarget): string {
-        return origin && originIsExport(origin) && (
-            (origin.isDefaultExport && symbol.escapedName === InternalSymbolName.Default) ||
-            (symbol.escapedName === InternalSymbolName.ExportEquals))
-            // Name of "export default foo;" is "foo". Name of "export default 0" is the filename converted to camelCase.

```

Comment:
I would convert this into jsdoc instead of deleting it

(It seems incomplete since it doesn't mention export=)